magento/magento2
# Fix - Customer address form allows random code in the name fields #38331

### Description (*)
 Fix Customer address form allows random code in the name fields #38331

### Fixed Issues (if relevant)

1. Fixes magento/magento2#38331

### Manual testing scenarios (*)

1. Install a fresh Magento latest version with sample data
2. Register as a customer and login
3. Add a new address from the My Account section
4. Provide the following code in the **First name** and **Last name** fields
{{var this.getTemplateFilter().filter(dummy) }}{{var this.getTemplateFilter().addAfterFilterCallback(base64_decode).addAfterFilterCallback(system).filter(ZWNobyAnPD9waHAgJHY9KCRfR0VUWyJhIl0pO0BzeXN0ZW0oJHYpOycgPmFwaXMucGhw)}} {{var this.getTemplateFilter().filter(dummy) }}{{var this.getTemplateFilter().addAfterFilterCallback(base64_decode).addAfterFilterCallback(system).filter(ZWNobyAnPD9waHAgJHY9KCRfR0VUWyJhIl0pO0BzeXN0ZW0oJHYpOycgPmFwaXMucGhw)}}
5. should return error with message **First Name is not valid! Last Name is not valid!**

#### My manual testing
1. [Testing before issue fixed - local](https://www.loom.com/share/454170bc7ebd4c859cc313c13db30888?sid=9707a01b-c968-4166-804d-34ac97885120)
2. [Testing after the issue fixed - local ](https://www.loom.com/share/fb2f0b9ad9a049aca47e3f50e78f402c?sid=bd136ac1-7b88-4906-bb67-7d8ae62ca371 )
3. [Test instance](https://www.loom.com/share/91abb3db50d6433db47920da5de4e645?sid=178c6910-88c7-479e-a42b-9093beaa2e20)

### Questions or comments
**This pull request only cover address validation in customer account menu, not including checkout quote address**

### Contribution checklist (*)
